App should fail when unable to connect to Registry

##### **Overview of the issue**

Microservices apps and gateways should fail when unable to connect to Registry

##### **Suggest a Fix** 

By setting this in bootstrap.yml

     spring.cloud.config.fail-fast: true

App will fail this way:

~~~
java.lang.IllegalStateException: Could not locate PropertySource and the fail fast property is set, failing
	at org.springframework.cloud.config.client.ConfigServicePropertySourceLocator.locate(ConfigServicePropertySourceLocator.java:110)

Caused by: org.springframework.web.client.ResourceAccessException: I/O error on GET request for "http://localhost:8761/config/srv1/dev/master": Connection refused; nested exception is java.net.ConnectException: Connection refused
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:607)

Caused by: java.net.ConnectException: Connection refused


~~~

--- a/generators/server/templates/src/main/resources/config/_bootstrap-dev.yml
+++ b/generators/server/templates/src/main/resources/config/_bootstrap-dev.yml
@@ -5,6 +5,7 @@
 spring:
     cloud:
         config:
+            fail-fast: true
             uri: http://localhost:8761/config
             # name of the config server's property source (file.yml) that we want to use
             name: <%= baseName %>

--- a/generators/server/templates/src/main/resources/config/_bootstrap-prod.yml
+++ b/generators/server/templates/src/main/resources/config/_bootstrap-prod.yml
@@ -5,6 +5,7 @@
 spring:
     cloud:
         config:
+            fail-fast: true
             #uri: http://localhost:8761/config
             # name of the config server's property source (file.yml) that we want to use
             name: <%= baseName %>

